### PR TITLE
[4.1] Add App Platform functions

### PR DESCRIPTION
**Pull request by shihjay2**
Created App Platform functions for DigitalOcean API v2

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "shihjay2/digitalocean-v2",
```
Comment (GrahamCampbell) [inline composer.json]: You should not publish forks on packagist. Instead, use composer's repositories feature to install your fork in your app.

**Comment by GrahamCampbell** [comment]
Thanks for working on this. :)

File: composer.json (hunk 1 above)
Comment (shihjay2) [reply composer.json]: Thanks, Graham for the feedback!  I was having trouble on my local testing to get my fork to work on my Laravel app only to realize later on that I needed to have a local fork of the Laravel DigitalOcean package have it recognize the local forked dependency - this got accidentally pushed out but it was corrected right away.

Comment (GrahamCampbell) [reply composer.json]: You don't have to fork the laravel package either. The composer repository feature allows you to drop your fork in as if it were the original package as published on packagist.

**Pull request merged**